### ISSUE TITLE
Cannot Connect Postgresql database to Grafana Dashboard using Postgres Exporter

### ISSUE THREAD

**Issue by paldapd** [prometheus-community/postgres_exporter#989]
Hello, I'm creating a dashboard database using Postgres Exporter, Prometheus, and Grafana. But it seems the database is not connected to the grafana dashboard because there is no data and the database not appear on dashboard. Then i check Prometheus, there is no query for pg_stat?
The only queries available are pg_exporter_last_scrape_duration_seconds, pg_up, pg_exporter_last_scrape_error, pg_exporter_scrapes_total.
So basically, I'm missing some metrics from the PostgreSQL exporter.
Can anyone help me on how to resolve this? Thank you.

![postgres database](https://github.com/prometheus-community/postgres_exporter/assets/79291811/9b8fee19-9875-4cf7-9f28-72307af8b453)
![prometheus postgres](https://github.com/prometheus-community/postgres_exporter/assets/79291811/6708bd1c-dd55-4690-8108-f4d949b950c6)

**Comment by vanallam** [comment]
I faced the same problem and I solved it after giving postgres_exporter the correct configs, so look at postgres_exporter logs it will help you
